.appveyor.yml

```diff
@@ -19,6 +19,9 @@ install:
   - ps: nuget install boost_serialization-vc140 -o "${env:APPVEYOR_BUILD_FOLDER}" -Version 1.60.0
   - ps: nuget install boost_math_c99-vc140 -o "${env:APPVEYOR_BUILD_FOLDER}" -Version 1.60.0
   - ps: nuget install OpenBLAS -o "${env:APPVEYOR_BUILD_FOLDER}"
+  - set path=C:\Program Files (x86)\WiX Toolset v3.11\bin;%path%
+  - ps: $ver = (Get-Content "${env:APPVEYOR_BUILD_FOLDER}\src\mlpack\core\util\version.hpp" | where {$_ -like "*MLPACK_VERSION*"}); $env:MLPACK_VERSION = ($ver[0].substring($ver[0].length-1,1)+'.'+$ver[1].substring($ver[1].length-1,1)+'.'+$ver[2].substring($ver[2].length-1,1))
```

Comment:
Hmm, the only thing is, if this is a git version instead of an official release, this will detect the most recently released version.  After the mlpack CMake configuration is done, the file `\src\mlpack\core\util\gitversion.hpp` will exist if this is the git master branch.  Internally will be just the single line

```
return "mlpack git-919dff2";
```

(Or whatever the git revision is.)  I don't speak powershell very well, but do you think it's possible to modify this to check if gitversion.hpp exists and extract `$ver` from there if so?

Reply:
Ah, oops, I forgot to respond to this.  The gitversion.hpp is generated at build time, so it will be available on AppVeyor.  We can handle this in a follow-up issue if needed.